libraries/AP_IQ_Motor/AP_IQ_Motor.cpp

```diff
@@ -105,34 +100,32 @@ uint8_t AP_IQ_Motor::find_client(ClientAbstract **set_client, uint8_t client_typ
 
 void AP_IQ_Motor::update(void)
 {
-    if (!_initialized)
-    {
+    //Only initialize once
+    if (!_initialized) {
         _initialized = true; // check if init worked
         init();
     }
 
-    if (iq_uart == nullptr)
-    {
+    //If we are ever unable to find an instance of our communication protocol, stop trying to update
+    if (iq_uart == nullptr) {
         return;
     }
 
     // TODO: generate general broadcast output message to esc prop from channels array
     // TODO: send telemetry request for current telemetry motor
```

Comment:
Same thing with these TODOs, are they planned to happen in PRs after this cosmetic one?